### PR TITLE
Add more metrics

### PR DESCRIPTION
**Pull request by MalazAlkoj**

### CODE REVIEW

File: src/main/kotlin/no/nav/syfo/metric/Metric.kt [hunk 1]
```diff
@@ -12,17 +12,22 @@ constructor(
     private val registry: MeterRegistry,
 ) {
     fun countSenOppfolgingSubmitted() = countEvent("sen_oppfolging_submitted")
+    fun countSenOppfolgingRequestYes() = countEvent("sen_oppfolging_requested_yes")
```
Comment (MalazAlkoj) [inline src/main/kotlin/no/nav/syfo/metric/Metric.kt:15]: When we receive `svar` with yes, regardless we managed to save it or not.

File: src/main/kotlin/no/nav/syfo/metric/Metric.kt [hunk 2]
```diff
@@ -12,17 +12,22 @@ constructor(
...
+    fun countSenOppfolgingRequestNo() = countEvent("sen_oppfolging_requested_no")
```
Comment (MalazAlkoj) [inline src/main/kotlin/no/nav/syfo/metric/Metric.kt:16]: When we receive `svar` with no, regardless we managed to save it or not.

File: src/main/kotlin/no/nav/syfo/metric/Metric.kt [hunk 3]
```diff
@@ -12,17 +12,22 @@ constructor(
...
+    fun countSenOppfolgingFerdigstilt() = countEvent("sen_oppfolging_ferdigstilt")
```
Comment (MalazAlkoj) [inline src/main/kotlin/no/nav/syfo/metric/Metric.kt:17]: After we actually register the answer

File: src/main/kotlin/no/nav/syfo/metric/Metric.kt [hunk 4]
```diff
@@ -12,17 +12,22 @@ constructor(
...
 
     fun countCallVeilarbregistreringComplete() = countEvent("call_veilarbregistrering_complete")
 
     fun countSenOppfolgingV2Submitted() = countEvent("sen_oppfolging_V2_submitted")
+    fun countSenOppfolgingV2SubmittedNo() = countEvent("sen_oppfolging_V2_submitted_no")
```
Comment (MalazAlkoj) [inline src/main/kotlin/no/nav/syfo/metric/Metric.kt:22]: Registered answer with `no`

**Review by luantr — APPROVED** [review]

**Pull request merged**